modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradePlusLayoutExperiment.kt

```diff
@@ -0,0 +1,185 @@
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import au.com.shiftyjelly.pocketcasts.account.viewmodel.OnboardingUpgradeFeaturesState
+import au.com.shiftyjelly.pocketcasts.compose.components.AutoResizeText
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP30
+import au.com.shiftyjelly.pocketcasts.compose.images.SubscriptionBadgeDisplayMode
+import au.com.shiftyjelly.pocketcasts.compose.images.SubscriptionBadgeForTier
+import au.com.shiftyjelly.pocketcasts.localization.R
+import au.com.shiftyjelly.pocketcasts.models.type.Subscription.SubscriptionTier
+import au.com.shiftyjelly.pocketcasts.settings.onboarding.OnboardingUpgradeSource
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+import au.com.shiftyjelly.pocketcasts.ui.R as UR
+
+@Composable
+internal fun UpgradePlusLayoutExperiment(
+    state: OnboardingUpgradeFeaturesState.Loaded,
+    source: OnboardingUpgradeSource,
+    scrollState: ScrollState,
+    onNotNowPressed: () -> Unit,
+    onFeatureCardChanged: (Int) -> Unit,
+    onClickSubscribe: () -> Unit,
+    canUpgrade: Boolean,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier.fillMaxHeight(),
+        contentAlignment = Alignment.BottomCenter,
+    ) {
+        BoxWithConstraints(
+            Modifier
+                .fillMaxHeight()
+                .background(Color.Black),
+        ) {
+            Box(modifier = Modifier.verticalScroll(scrollState)) {
+                Column(
+                    Modifier
+                        .windowInsetsPadding(WindowInsets.statusBars)
+                        .windowInsetsPadding(WindowInsets.navigationBars)
+                        .heightIn(min = this@BoxWithConstraints.calculateMinimumHeightWithInsets())
+                        .padding(bottom = 100.dp),
+                ) {
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(top = 8.dp),
+                        horizontalArrangement = Arrangement.End,
+                    ) {
+                        TextP30(
+                            text = stringResource(R.string.not_now),
+                            color = Color.White,
+                            modifier = Modifier
+                                .padding(start = 24.dp, end = 24.dp, top = 8.dp)
+                                .clickable { onNotNowPressed() },
```

Comment:
Would it be good to use a button so we get a press state?

Reply:
I changed to use the existing RowTextButton component instead [0d0329c](https://github.com/Automattic/pocket-casts-android/pull/2635/commits/0d0329cec0372ed1269ac77053b296c5cd48c4b0)